### PR TITLE
Add DB test data

### PR DESCRIPTION
**Pull request by jeon0121**
1. I think `OID` shouldn't be placed in `Discount` table. Instead, `discount ID` should be placed in `Order`. Also total participation between `Discount` and `Order` in the `ER diagram` might be incorrect? not sure, you can think about it
2. If `order time` change from `timestamp` to `datetime`, tell me and I would modify test data
3. If `SQL type` need to be modified, pls also update the `SQL command` in our google document

### CODE REVIEW

**Comment by ilovemath25** [comment]
you right, since order can apply discount, but not all discount dont get applied by order.

**Comment by ilovemath25** [comment]
Screenshot_20241129_072722_LINE

i think it's okay if we change it like this (partial : partial)

**Comment by jeon0121** [comment]
> Screenshot_20241129_072722_LINE
> 
> i think it's okay if we change it like this (partial : partial)

agree

File: DB_testdata.sql [hunk 1]
```diff
@@ -0,0 +1,88 @@
+INSERT INTO "Member" ("F_name", "L_name", "Password", "Gender", "Email", "Phone", "Birth", "Address", "Reg_date", "A_flag", "S_flag", "C_flag") 
+VALUES 
+('Harrison', 'Mckinney', 'Password123', 'Male',   'harrison4356@gmail.com',  '0912345678',  '1990-01-15', '123 Main St, Springfield',   '2024-10-13', TRUE,  FALSE, FALSE),
+('Hudson',   'Smith',    'Passw0rd!',   'Male',   'hudson247@gmail.com',     '0923456789',  '1992-05-20', '456 Elm St, Springfield',    '2024-10-14', FALSE, TRUE,  FALSE),
+('Chris',    'Lee',      'Qwerty123',   'Other',  'chris_lee@gmail.com',      NULL,         '1995-03-10', '789 Maple St, Springfield',  '2024-10-28', FALSE, FALSE, TRUE),
```
Comment (ilovemath25) [inline DB_testdata.sql:5]: phone number is NOT NULL
Please change

File: DB_testdata.sql [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+INSERT INTO "Discount" ("Disc_code", "Disc_value", "Disc_type", "Disc_name", "Policy_desc", "Max_usage")
+VALUES
+('SHIP500FREE', 0.00, 'Shipping', 'Free Shipping Over $500', 
+ 'Applies to orders with a total purchase amount greater than or equal to $500. Valid from 2024-11-01 to 2024-12-31.', 
+ 1000),
+('SHIP10OFF', 0.90, 'Shipping', 'Flat 10% Off Shipping Fee', 
+ 'Valid for orders placed from 2024-11-15 to 2024-12-31. Shipping fee will be reduced by 10% for all eligible orders.', 
+ 500),
+('SHIPVIP', 0.00, 'Shipping', 'VIP Free Shipping',
```
Comment (ilovemath25) [inline DB_testdata.sql:46]: we dont have VIP
please change

File: DB_testdata.sql [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+('4532147890123456', 3,  '2025-12-31', '123'),
+('5312345678901233', 4,  '2026-06-30', '456'), 
+('3530111333300505', 5,  '2027-09-15', '789'),
+('5412345678901232', 6,  '2026-03-20', '321'),
+('5212345678901234', 10, '2028-11-10', '654');
+
+
+INSERT INTO "Order" ("CMID", "SMID", "Credit_num", "Time", "Ship_address", "Ship_fee", "Status", "Pay_method", "Tot_price") 
+VALUES
+(10, 9, '5212345678901234', '2024-11-24 09:30:00', '987 Elm Dr, Brockway',           35, 'Shipping',   'Credit card', 1300),
```
Comment (ilovemath25) [inline DB_testdata.sql:26]: Do you think we should use Taiwan address?
If want, you can change it

**Review by ilovemath25 — CHANGES_REQUESTED** [review]

File: DB_testdata.sql (hunk 3 above)
Comment (jeon0121) [reply DB_testdata.sql]: No, i think this is fine. or maybe you can ask others opinion in meeting tmr
also i add the `DID` in `Order` and fix problems you have mentioned above

File: DB_testdata.sql [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+VALUES
+('SHIP500FREE', 0.00, 'Shipping', 'Free Shipping Over $500', 
+ 'Applies to orders with a total purchase amount greater than or equal to $500. Valid from 2024-11-01 to 2024-12-31.', 
+ 1000),
+('SHIP10OFF', 0.90, 'Shipping', 'Flat 10% Off Shipping Fee', 
+ 'Valid for orders placed from 2024-11-15 to 2024-12-31. Shipping fee will be reduced by 10% for all eligible orders.', 
+ 500),
+('FIRSTSHIP', 0.00, 'Shipping', 'Free Shipping for First Order', 
+ 'Applies to first-time customers on their first order, regardless of purchase amount. Valid from 2024-10-01 to 2024-12-31.', 
+ 2000),
```
[the remark this answers is not in the document]
Comment (jeon0121) [reply DB_testdata.sql:48]: this is for first-time orders, every customer who registers an account can use one, so i think this should be large enough to cover this

**Review by ilovemath25 — APPROVED** [review]

**Pull request merged**